pkg/datapath/linux/node_ids.go

```diff
@@ -23,6 +23,13 @@ const (
 	maxNodeID = idpool.ID(^uint16(0))
 )
 
+var (
+	localNodeID = nodeID{
+		id:     0,
+		refcnt: 1,
```

Comment:
Is this refcount here ever increased/decreased, or is this basically a constant?

If the contents can here (i.e. if this is stateful), then we should not make this a global variable and instead have it be part of the `linuxNodeHandler`

Reply:
No refcnt never changes its just a static id so I had something to say was the localNode.

Reply:
I guess golang does not understand const structs.